content/graphql/resolvers-map.md

```diff
@@ -396,10 +476,10 @@ Once we're done with the above steps, we have to provide the `AuthorResolver` so
 export class AuthorsModule {}
 ```
 
-The `GraphQLModule` will take care of reflecting the metadata and transforming classes into the correct resolvers map automatically. The only thing you need to be aware of is that you need to import this module somewhere, so Nest will be able to utilize `AuthorsModule`.
-
 > info **Hint** Learn more about GraphQL queries [here](http://graphql.org/learn/queries/).
```

Comment:
We should probably move this hint elsewhere (maybe even duplicate in the code first and schema first sections(?))